### PR TITLE
Fix defaultSortFieldName and defaultSortDirection pagination option from KNP

### PR DESCRIPTION
**Pull request by marydn**
Apply defaultSortFieldName and defaultSortDirection pagination options if they are defined. Fix for previous PR.

https://github.com/FriendsOfSymfony/FOSElasticaBundle/pull/1032#issuecomment-191746589

### CODE REVIEW

**Comment by XWB** [comment]
@dariuszz123 @robhunt3r Can you guys check this out?

**Comment by robhunt3r** [comment]
Working for me, it checks if key exists before using so no crashing anywhere.

File: Subscriber/PaginateElasticaQuerySubscriber.php [hunk 1]
```diff
@@ -60,12 +60,21 @@ public function items(ItemsEvent $event)
     protected function setSorting(ItemsEvent $event)
     {
         $options = $event->options;
-        $sortField = $this->request->get($options['sortFieldParameterName'], $options['defaultSortFieldName']);
+        $sortField = $this->request->get($options['sortFieldParameterName']);
+
+        if(!$sortField && isset($options['defaultSortFieldName'])) {
```
Comment (XWB) [inline Subscriber/PaginateElasticaQuerySubscriber.php]: Missing space, should be `if (`

File: Subscriber/PaginateElasticaQuerySubscriber.php [hunk 2]
```diff
@@ -60,12 +60,21 @@ public function items(ItemsEvent $event)
...
+            $sortField = $options['defaultSortFieldName'];
+        }
 
         if (!empty($sortField)) {
             // determine sort direction
             $dir = 'asc';
-            $sortDirection = $this->request->get($options['sortDirectionParameterName'], $options['defaultSortDirection']);
+            $sortDirection = $this->request->get($options['sortDirectionParameterName']);
+            
+            if(!$sortDirection && isset($options['defaultSortDirection'])) {
```
Comment (XWB) [inline Subscriber/PaginateElasticaQuerySubscriber.php]: Missing space, should be `if (`

**Comment by XWB** [comment]
@marydn There seems to be a merge conflict, can you rebase from master?

**Comment by XWB** [comment]
Something is still wrong, now there are 9 commits?

**Comment by XWB** [comment]
I fixed the merge conflicts https://github.com/FriendsOfSymfony/FOSElasticaBundle/pull/1050

**Pull request closed without merging**